api-client/src/test/java/com/xing/api/CallSpecTest.java

```diff
@@ -1206,11 +1211,69 @@ public void canOverrideReadAndConnectTimeoutPerCall() throws Exception {
     }
 
     @Test
-    public void disallowZeroOrNegativeTimeouts() throws Exception {
+    public void canOverrideWriteTimeoutPerCall() throws Exception {
+        // for some unknown reason the timeout is not applied properly when run on Travis CI
+        // so we skip this test on the CI and only execute it locally
+        assumeThat(System.getenv("TRAVIS"), not(equalTo("true")));
```

Comment:
You shouldn't do that, when discussing with @yayaa we saw that you are using `System.currentTimeMillis()` which returns the time of the system obviously. I checked the time and saw 

```
Build system information
Build language: android
Build group: stable
Build dist: precise
Build id: 203917366
Job id: 203917367
travis-build version: 6fa97f65a
Build image provisioning date and time
Thu Feb  5 15:09:33 UTC 2015
Operating System Details
Distributor ID:	Ubuntu
Description:	Ubuntu 12.04.5 LTS
Release:	12.04
Codename:	precise
```

So for some reason travis is stuck in 2015 which returns the wrong timestamp then.
I could reproduce this on my own machine setting the time to the year 2015 I got the following error:

```
INFO: MockWebServer[53225] connection from /127.0.0.1 failed: java.net.SocketException: Socket closed

java.lang.AssertionError: 
Expecting:
 <1291L>
to be less than:
 <1200L> 

	at com.xing.api.CallSpecTest.canOverrideWriteTimeoutPerCall(CallSpecTest.java:1252)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:239)
	at okhttp3.mockwebserver.MockWebServer$2.evaluate(MockWebServer.java:154)
	at com.xing.api.RecordingSubscriber$Rule$1.evaluate(RecordingSubscriber.java:149)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:68)
	at com.intellij.rt.execution.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:51)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:237)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:70)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
```

which is exactly the same one that travis showed. So please use constants instead in order to fix this test on travis.